src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -104,26 +110,22 @@ private void ShouldAllowNavigation(object sender, CoreWebView2NavigationStarting
         /// <param name="link"></param>
         public void NavigateToPage(Uri link)
         {
-            InitializeAsync();
+            Dispatcher.Invoke(InitializeAsync);
```

Comment:
is Dispatcher the UI thread dispatcher here?

Reply:
yeah, should this be BeginInvoke?

Reply:
maybe it is not obvious enough, this is the Dispatcher of the current UI control (this)
All UI elements share the same dispatcher and that is the UI thread's dispatcher.